applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/images/mcuboot/prj_release_dongle.conf

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_SIZE_OPTIMIZATIONS=y
+
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_MAIN_STACK_SIZE=10240
+CONFIG_BOOT_BOOTSTRAP=n
+
+CONFIG_BOOT_VERSION_CMP_USE_BUILD_NUMBER=y
+
+CONFIG_FLASH=y
+# CONFIG_FPROTECT is not supported yet on the nRF54H20 SoC
+
+CONFIG_RESET_ON_FATAL_ERROR=y
+
+# Configure Zephyr system power management
+# The Zephyr system power management is used by the MCUboot bootloader to detect wake-up from
+# S2RAM and redirect execution to the resume routine of the application image
+CONFIG_PM=y
+CONFIG_PM_S2RAM=y
+CONFIG_PM_S2RAM_CUSTOM_MARKING=y
+
+# Enable HW cryptography in the MCUboot bootloader and its dependencies
+# The PSA operations are handled by the SDFW Service Framework (SDFW)
+# that is enabled with the CONFIG_PSA_SSF_CRYPTO_CLIENT Kconfig option.
```

Comment:
I would consider explicitly enabling the `CONFIG_PSA_SSF_CRYPTO_CLIENT` instead of commenting on that (to ensure it's actually set here). I also wonder if we actually need keep the `CONFIG_PSA_CRYPTO_DRIVER_OBERON` enabled. Do we actually use the driver?